tests/Test/TestSuiteSpec.hs

```diff
@@ -64,16 +63,6 @@ spec = describe "Souffle API" $ parallel $ do
       edges `shouldBe` [Edge "b" "c", Edge "a" "b"]
       reachables `shouldBe` [Reachable "b" "c", Reachable "a" "c", Reachable "a" "b"]
 
-    it "can retrieve facts as a vector" $ do
```

Comment:
As I removed the `CollectFacts` type this test case had to go.

Reply:
Hmm 🤔 I hope I can re-add it again 😝